packages/nc-gui/components/virtual-cell/components/ListChildItems.vue

```diff
@@ -124,23 +130,21 @@ onKeyStroke('Escape', () => {
   vModel.value = false
 })
 
-const skeltonAmountToShow = ref(childrenListCount.value === 0 ? 10 : childrenListCount.value)
-
 /* 
    to render same number of skelton as the number of cards
    displayed
  */
-watch(childrenListPagination, () => {
+
+const skeltonCount = computed(() => {
   if (childrenListCount.value < 10 && childrenListPagination.page === 1) {
-    skeltonAmountToShow.value = childrenListCount.value === 0 ? 10 : childrenListCount.value
+    return childrenListCount.value === 0 ? 10 : childrenListCount.value
```

Comment:
```ts
return childrenListCount.value || 10
```

Reply:
one doubt  , how does this work ? 

if   ``` childrenListCount.value ``` === 0 it will return 0 right ? but we need 10 there.

Reply:
> one doubt , how does this work ?

`||` is OR operator, which uses the right value if left one is [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy). if `a === 0`, then `a || 10 -> 10`.
On the other hand, the nullish coalescing operator `??` uses the right value if left one is `null` or `undefined`. If `a === 0`, then `a ?? 10 -> 0`. If `a === null / undefined`, then `a ?? 10 -> 10`.

> if childrenListCount.value === 0 it will return 0 right ? but we need 10 there.

It returns 10.